web/src/app/auto-count/auto-count-dialog.component.html

```
@@ -0,0 +1,46 @@
+<h2 mat-dialog-title>Liegestütze automatisch zählen</h2>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)